.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: dev_updated
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the dev branch
+on:
+  push:
+    branches: [ dev ]
```

Comment:
Is there a way we can restrict it to only run on merge commits for PRs? Otherwise how many builds will be kicked off if a PR with n commits merges into dev? Will it be `n + 1` builds kicked off at once?